styleguide/derek/examples/office365.jade

```diff
@@ -0,0 +1,4 @@
+extends ../../_layouts/page
+
+block content
+  include ./_office365.html
```

Comment:
@amberRrucker you can add the `_mainnav` partial here as with solutions:

https://github.com/rackerlabs/zoolander/blob/492e39cc8c8600a9e620602aaa3eca0f477d0dda/styleguide/derek/examples/solutions.jade#L4